redis/tests/test_basic.rs

```diff
@@ -1280,3 +1280,32 @@ fn test_set_options_options() {
 
     assert_args!(&opts, "EX", "1000");
 }
+
+#[test]
+fn test_blocking_sorted_set_api() {
+    let ctx = TestContext::new();
+    let mut con = ctx.connection();
+
+    assert_eq!(con.zadd("a", "1a", 1), Ok(()));
+    assert_eq!(con.zadd("b", "2b", 2), Ok(()));
+    assert_eq!(con.zadd("c", "3c", 3), Ok(()));
+    assert_eq!(con.zadd("d", "4d", 4), Ok(()));
+    assert_eq!(con.zadd("a", "5a", 1), Ok(()));
+    assert_eq!(con.zadd("b", "6b", 2), Ok(()));
+    assert_eq!(con.zadd("c", "7c", 3), Ok(()));
+    assert_eq!(con.zadd("d", "8d", 4), Ok(()));
+
+    let result_multiple_min = con
```

Comment:
Did you mean to only test the `min` command variants?

Reply:
You're right, the max variables should have been the result of max functions. My mistake. 
I fixed it in [a08aba1](https://github.com/redis-rs/redis-rs/pull/962/commits/a08aba18f1b2ac8ebf50e7328a271761f0048c6a).